### DISCUSSION TITLE
Takes long to access github on terminal

### DISCUSSION THREAD

**Discussion by decipher3114** [community/community#38331] [Other Feature Feedback, Questions, & Ideas] [answered]
I just got a new wifi-router installed and sadly enough, the ISP doesn't provide IPv6.

The problem I am facing is I can open github on browser and use it. It works normally.

But it took about 30 sec to connect to github on terminal.

Clearly, if I want to fetch a release package from github of size, say 100MB, then it would take first 25-30 secs to connect to github and then another 10 secs to download the file.

I've tried disabling Ipv6 on Windows and WSL(Ubuntu) as well.
So, Only takes time while connecting.
Anyone have idea about what could be the problem and how to solve it.

**Comment by decipher3114** [comment, marked as answer]
Fixed by changing dns

**Reply by gaurav517** [reply]
Can you please elaborate? Did you change your dns server? Thanks.
